src/global/components/Layout/index.jsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const index = () => {
+  return <div className="bg-d-medium"></div>;
```

Comment:
not sure, but maybe you need to enter {children} as placeholder here? like
`function DefaultLayout({ children, className }) {
    return (
        <div className="bg-dark h-auto w-100 ${className}">
            { children }
            <NavBar/>
        </div>
    );
}`